analysis/test/src/pointers.rs

```diff
@@ -413,6 +413,17 @@ pub unsafe extern "C" fn test_realloc_fresh() {
 #[no_mangle]
 pub unsafe extern "C" fn test_load_addr() {
     let s = malloc(::std::mem::size_of::<S>() as libc::c_ulong) as *mut S;
+    *s = S {
+        field: 0i32,
+        field2: 0u64,
+        field3: 0 as *const S,
+        field4: T {
+            field: 0i32,
+            field2: 0u64,
+            field3: 0 as *const S,
+            field4: 0i32,
+        },
+    };
```

Comment:
See comment by @aneksteind: https://github.com/immunant/c2rust/commit/f6e5bce29673ff7170a9cec704fcbce925a32f2a#r85154562.

Reply:
`calloc` works, so fixed in https://github.com/immunant/c2rust/pull/683/commits/f21764b3b4a82cbf9da04cc088cba6a5a6b2a5c2.